ckanext/stadtzhtheme/plugin.py

```diff
@@ -272,7 +272,8 @@ def update_config(self, config):
 
         tk.add_template_directory(config, 'templates')
         tk.add_public_directory(config, 'public')
-        tk.add_resource('fanstatic', 'stadtzhtheme')
+        tk.add_resource('assets', 'stadtzh_theme')
```

Comment:
what does `stadtzh_theme` (without any ending) refer to? Haven't found it anywhere else, or is it just the theme-name and loading assets that way?

Reply:
d'oh, you're right, that should be `stadtzh_theme_css`. Will fix it.